mula/.ci/docker-compose.yml

```diff
@@ -24,20 +24,23 @@ services:
     env_file:
       - .ci/.env.test
     depends_on:
-      ci_postgres:
-        condition: service_healthy
-      ci_rabbitmq:
-        condition: service_healthy
+      - ci_postgres
+      - ci_bytes
+      - ci_katalogus
+      - ci_rabbitmq
 
   ci_postgres:
     image: postgres:15
     healthcheck:
-      test: ["CMD", "pg_isready"]
+      test: ["CMD", "gosu", "postgres", "pg_isready"]
       interval: 10s
```

Comment:
Since the timeout is removed, I suggest using a lower interval (with a possibly higher retry count)